plat/drivers/include/uart/com.h

```diff
@@ -19,11 +19,11 @@
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#ifndef __KVM_SERIAL_CONSOLE__
-#define __KVM_SERIAL_CONSOLE__
+#ifndef __COM_H__
+#define __COM_H__
 
 void _libkvmplat_init_serial_console(void);
```

Comment:
COMx devices are not KVM specific: https://en.wikipedia.org/wiki/COM_(hardware_interface), so the interface provided by this driver should come with appropriate names. Also shouldn't this driver also provide a set of ops like the rest?